CONTRIBUTING.md

```diff
@@ -140,3 +140,24 @@ your feature works as intended.
 After this point, you may be asked to make modification to your code to adhere
 to coherent code style or to fix bugs you may have not noticed. Once you get
 approvals from 1 or 2 repository owners, we will merge your code! :confetti_ball: :tada:
+
+
+### Using a fork
+
+If you're developing on a fork of Blueprint, navigate to
+[CircleCI](https://circleci.com/), log in using your GitHub account and enable
```

Comment:
nit: I'm sure our docs aren't consistent about this, but I'd like to see an [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) here:

```
...your GitHub account, and enable...
```